client/components/list-end/docs/example.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ListEnd from 'components/list-end';
+
+export default function ListEndExample() {
+	return <ListEnd />;
+}
+ListEndExample.displayName = 'ListEndExample';
```

Comment:
It's recommended in the [readme](https://github.com/Automattic/wp-calypso/tree/ef86b5b8fb324a5a030747c6df5ecdef6df50eab/client/devdocs/design#component-name-convention). I believe because keeps a readable name for devtools in production builds.